src/components/BrowserCompat/BrowserCompat.module.css

```diff
@@ -0,0 +1,144 @@
+.browserCompat {
```

Comment:
@AlexLakatos this part makes it fat now

Reply:
But also, now the legend isn't hidden with the rest of the component. 
![Uploading Screenshot 2023-07-17 at 15.48.31.png…]()